src/formApps/_ItemMenu/components/category/CategoryControl.svelte

```diff
@@ -58,7 +58,8 @@
 
   //Check the Autorec Menu for a matching Section
   let filteredSettings = AAAutorecFunctions.sortAndFilterMenus(autorecSettings)
-  $: isInAutorec = AAAutorecFunctions.allMenuSearch(filteredSettings, AAAutorecFunctions.rinseName($animation.label), $animation.label);
+  const realName = AAAutorecFunctions.getRealName($animation.label, item);
```

Comment:
This will likely never update due to realName being a const. This is a svelte file, with reactivity being key. 
For simplicity's sake, try putting the getRealName into the function call itself so it is executed with it as well.

Reply:
Not sure it's a good idea to have getRealName into allMenuSearch/singleMenuSearch. I would need to change the signature to pass the item and it's not really the role of a search function to provide the right name